### PR TITLE
#8 Maya v1.0.0 Locators docs

### PR DESCRIPTION
**Pull request by lucasferryinbibo**
# Description of this issue
Locators documented.

# Pages to review
locators.md

Issue of this PR: https://github.com/Inbibo/adonisfx_docs/issues/8

### CODE REVIEW

**Comment by lucasferryinbibo** [comment]
Appoved on my behalf as I created the PR

File: docs/documentation/maya/locators.md [hunk 1]
```diff
@@ -1,113 +1,138 @@
...
 
- 1. Select three scene objects with transform nodes. The order in which the objects are selected is relevant, as the created angle will have the following arrangement:
-    - First selected object: end point of angle.
-    - Second selected object: middle point of angle.
-    - Third selected object: end pont of angle.
- 2. Press the ![AdnLocatorRotation button](images/adn_angle_locator.png) button in the AdonisFX shelf or press *Rotation* in the AdonisFX menu, under the *Locators* submenu.
- 3. The AdnLocatorRotation is created and ready to be used.
+#### Input
+ - **Start Position**(Float3, {0.0. 0.0, 0.0}): Start transform node position.
+ - **End Position**(Float3, {0.0. 0.0, 0.0}): End transform node position.
```
Comment (josevalverdeinbibo) [inline docs/documentation/maya/locators.md:68]:
```suggestion
 - **Start Position** (Float3, {0.0. 0.0, 0.0}): Start transform node position.
 - **End Position** (Float3, {0.0. 0.0, 0.0}): End transform node position.
```

File: docs/documentation/maya/locators.md [hunk 2]
```diff
@@ -1,113 +1,138 @@
...
+ - **Velocity** (Float, 0.0): Magnitude of the velocity (remapped) between the transform nodes.
+ - **Acceleration** (Float, 0.0): Magnitude of the acceleration (remapped) between the transform nodes.
+
+#### Draw
+ - **Scale** (Float, 1.0): Sets the scaling factor applied to the distance locator visualizer.
+    - Has a range of \[0.0, 10.0\]. Upper limit soft, higher values can be used.
+ - **Draw Output** (Enumerator, "Distance"): Selects the property of the locator to be visualized on the locator visualizer.
+    - **Distance:** color the visualizer of the locator according to the input distance activation.
+    - **Velocity:** color the visualizer of the locator according to the input velocity activation.
+    - **Acceleration:** color the visualizer of the locator according to the input acceleration activation.
```
Comment (josevalverdeinbibo) [inline docs/documentation/maya/locators.md:81]:
```suggestion
    - **Distance:** Color the visualizer of the locator according to the input distance activation.
    - **Velocity:** Color the visualizer of the locator according to the input velocity activation.
    - **Acceleration:** Color the visualizer of the locator according to the input acceleration activation.
```

File: docs/documentation/maya/locators.md [hunk 3]
```diff
@@ -1,113 +1,138 @@
...
+### Create AdnLocatorRotation
 
-All AdnLocator types share similar attributes, with some specific differences on information displayed. Locator Attributes are all grouped together here but differences will be pointed out.
+Three transform nodes will be required to create the AdnLocatorRotation. The creation process is the following:
 
-[^1]:  Soft range: higher values can be used.
+ 1. Select three scene objects with transform nodes. The order in which the objects are selected is relevant, as the created angle will have the following arrangement:
+    - First selected object: start point of the angle.
+    - Second selected object: middle point of the angle.
+    - Third selected object: end point of the angle.
```
Comment (josevalverdeinbibo) [inline docs/documentation/maya/locators.md:103]:
```suggestion
    - First selected object: Start point of the angle.
    - Second selected object: Middle point of the angle.
    - Third selected object: End point of the angle.
```

File: docs/documentation/maya/locators.md [hunk 4]
```diff
@@ -1,113 +1,138 @@
...
+ 2. Press the ![AdnLocatorRotation button](images/adn_angle_locator.png) button in the AdonisFX shelf or press *Rotation* in the AdonisFX menu, under the *Locators* submenu. If the shelf button is double-clicked or the option box in the menu is selected a window will be displayed where a custom name.
```
Comment (josevalverdeinbibo) [inline docs/documentation/maya/locators.md:104]:
```suggestion
 2. Press the ![AdnLocatorRotation button](images/adn_angle_locator.png) button in the AdonisFX shelf or press *Rotation* in the AdonisFX menu, under the *Locators* submenu. If the shelf button is double-clicked or the option box in the menu is selected a window will be displayed where a custom name and initial attribute values can be set.
```

File: docs/documentation/maya/locators.md [hunk 5]
```diff
@@ -1,113 +1,138 @@
...
+
+### Attributes
 
 #### Input
- - **Position**(float3): Current transform node position.
-    - In the case of AdnLocatorDistance, 2 positions will be displayed, one for each transform node. 
-    - In the case of AdnLocatorRotation, 3 positions will be displayed, one for each transform node. 
+ - **Start Position**(Float3, {0.0. 0.0, 0.0}): Start transform node position.
+ - **Mid Position**(Float3, {0.0. 0.0, 0.0}): Mid transform node position.
+ - **End Position**(Float3, {0.0. 0.0, 0.0}): End transform node position.
```
Comment (josevalverdeinbibo) [inline docs/documentation/maya/locators.md:112]:
```suggestion
 - **Start Position** (Float3, {0.0. 0.0, 0.0}): Start transform node position.
 - **Mid Position** (Float3, {0.0. 0.0, 0.0}): Mid transform node position.
 - **End Position** (Float3, {0.0. 0.0, 0.0}): End transform node position.
```

File: docs/documentation/maya/locators.md [hunk 6]
```diff
@@ -1,113 +1,138 @@
...
-    - This attribute is only available for AdnLocatorRotation.
- - **Velocity** (Float): Displays the transform's velocity magnitude.
- - **Acceleration** (Float): Displays the transform's acceleration magnitude.
+#### Draw
+ - **Scale** (Float, 1.0): Sets the scaling factor applied to the rotation locator visualizer.
+    - Has a range of \[0.0, 10.0\]. Upper limit soft, higher values can be used.
+ - **Draw Output** (Enumerator, "Angle"): Selects the property of the locator to be visualized on the locator visualizer.
+    - **Angle:** color the visualizer of the locator according to the input angle activation.
+    - **Velocity:** color the visualizer of the locator according to the input velocity activation.
+    - **Acceleration:** color the visualizer of the locator according to the input acceleration activation.
```
Comment (josevalverdeinbibo) [inline docs/documentation/maya/locators.md:125]:
```suggestion
    - **Angle:** Color the visualizer of the locator according to the input angle activation.
    - **Velocity:** Color the visualizer of the locator according to the input velocity activation.
    - **Acceleration:** Color the visualizer of the locator according to the input acceleration activation.
```

File: docs/documentation/maya/locators.md [hunk 7]
```diff
@@ -1,113 +1,138 @@
...
+## AdnLocator
 
-## AdonisFX Logo locator
+The AdnLocator is an native alternative to Maya locators. This locator can be used to visualize any kind of scene element with a transform node. For example they can be used to visualize [attachments](muscle.md#attachments) or [segment ends](muscle.md#slide-on-segment-constraint) in muscle deformers.
 
-The AdonisFX Logo locator is an Adonis native alternative to Maya locators. These can be used as any kind of scene element with a transform node, be it to use to gather locator information or as [attachments](muscle.md#attachments) or [segment ends](muscle.md#slide-on-segment-constraint) in muscle deformers.
+### Create AdnLocator
 
-To create an AdonisFX Logo locator just click on the ![AdonisFX Logo locator button](images/adn_adonis_locator.png) button in the AdonisFX shelf. The locator will be created at the origin point of your scene.
+To create an AdonisFX Logo locator just click on the ![AdonisFX Logo locator button](images/adn_adonis_locator.png) button in the AdonisFX shelf. The locator will be created at the origin of your scene.
```
Comment (josevalverdeinbibo) [inline docs/documentation/maya/locators.md:133]:
```suggestion
To create an AdnLocator just click on the ![AdonisFX Logo locator button](images/adn_adonis_locator.png) button in the AdonisFX shelf. The locator will be created at the origin of your scene.
```

File: docs/documentation/maya/locators.md [hunk 8]
```diff
@@ -1,113 +1,138 @@
...
+### Attributes
 
-In this specific case the required sensor will be an [AdnSensorPosition](sensors.md).
-
-<figure markdown>
-  ![AdnLocatorPosition within a scene](images/locators_position.png){width=60%}
-  <figcaption>Figure 1: Position locator used in a human model.</figcaption>
-</figure>
+#### Input
+ - **Position**(Float3, {0.0. 0.0, 0.0}): Current transform node position.
```
[the remark this answers is not in the document]
Comment (josevalverdeinbibo) [reply docs/documentation/maya/locators.md:30]: Done.

File: docs/documentation/maya/locators.md [hunk 9]
```diff
@@ -1,113 +1,138 @@
...
+ - **Velocity** (Float, 0.0): Magnitude of the velocity (remapped) of the transform node.
+ - **Acceleration** (Float, 0.0): Magnitude of the acceleration (remapped) of the transform node.
 
-AdnLocatorDistance is the Adonis locator for visualizing the relative distance, velocity and acceleration between two transform nodes.
+#### Draw
+ - **Scale** (Float, 1.0): Sets the scaling factor applied to the position locator visualizer.
+    - Has a range of \[0.0, 10.0\]. Upper limit soft, higher values can be used.
+ - **Draw Output** (Enumerator, "Velocity"): Selects the property of the locator to be visualized on the locator visualizer.
+    - **Velocity:** color the visualizer of the locator according to the input velocity activation.
+    - **Acceleration:** color the visualizer of the locator according to the input acceleration activation.
```
[the remark this answers is not in the document]
Comment (josevalverdeinbibo) [reply docs/documentation/maya/locators.md:41]: Done.

File: docs/documentation/maya/locators.md [hunk 10]
```diff
@@ -1,113 +1,138 @@
...
 
-## AdnLocatorRotation
+### Create AdnLocatorDistance
 
-AdnLocatorRotation is the Adonis locator for visualizing the angle, angular velocity and angular acceleration between three transform nodes.
+Two transform nodes will be required to create an AdnLocatorDistance representing each extremity. The creation process is the following:
 
-### Create AdnLocatorRotation
+ 1. Select two scene objects with transform nodes.
+ 2. Press the ![AdnLocatorDistance button](images/adn_distance_locator.png) button in the AdonisFX shelf or press *Distance* in the AdonisFX menu, under the *Locators* submenu. If the shelf button is double-clicked or the option box in the menu is selected a window will be displayed where a custom name.
```
[the remark this answers is not in the document]
Comment (josevalverdeinbibo) [reply docs/documentation/maya/locators.md:61]: Done.

File: docs/documentation/maya/locators.md [hunk 11]
```diff
@@ -1,113 +1,138 @@
...
+</figure>
 
 ### Create AdnLocatorPosition
 
-Only one transform will be required to create the AdnLocatorPosition. The process that we must follow to do so is as following:
+Only one transform will be required to create the AdnLocatorPosition. The creation process is the following:
 
  1. Select a scene object with a transform node.
- 2. Press the ![AdnLocatorPosition button](images/adn_point_locator.png) button in the AdonisFX shelf or press *Position* in the AdonisFX menu, under the *Locators* submenu.
+ 2. Press the ![AdnLocatorPosition button](images/adn_point_locator.png) button in the AdonisFX shelf or press *Position* in the AdonisFX menu, under the *Locators* submenu. If the shelf button is double-clicked or the option box in the menu is selected a window will be displayed where a custom name.
```
[the remark this answers is not in the document]
Comment (josevalverdeinbibo) [reply docs/documentation/maya/locators.md:23]: Done.

File: docs/documentation/maya/locators.md (hunk 1 above)
Comment (josevalverdeinbibo) [reply docs/documentation/maya/locators.md:68]: Done.

File: docs/documentation/maya/locators.md (hunk 2 above)
Comment (josevalverdeinbibo) [reply docs/documentation/maya/locators.md:81]: Done.

File: docs/documentation/maya/locators.md (hunk 3 above)
Comment (josevalverdeinbibo) [reply docs/documentation/maya/locators.md:103]: Done.

File: docs/documentation/maya/locators.md (hunk 5 above)
Comment (josevalverdeinbibo) [reply docs/documentation/maya/locators.md:112]: Done.

File: docs/documentation/maya/locators.md (hunk 7 above)
Comment (josevalverdeinbibo) [reply docs/documentation/maya/locators.md:133]: Done.

**Review by josevalverdeinbibo — APPROVED** [review]

**Pull request merged**